.github/workflows/roxctl.yml

```diff
@@ -0,0 +1,33 @@
+name: Roxctl Image Scan
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+
+permissions:
+  contents: read
+
+jobs:
+  roxctl:
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
```

Comment:
Is this necessary? It seems like there is no `codeql-action`.

Reply:
Oh, no it's not needed. I though we have sarif format support